Cargo.toml

```diff
@@ -17,15 +17,13 @@ smallvec = "1.11.2"
 rustc-hex = "2.1.0"
 env_logger = "0.10.1"
 lru = "0.12.1"
-parking_lot = "0.12.1"
-
-bitvec = { version = "1", default-features = false }
+bitvec = { version = "1" }
 derive_more = { version = "0.99.17", features = ["constructor"] }
-mp-felt = { git = "https://github.com/keep-starknet-strange/madara.git", rev = "f30acea8af7e28e956e771928130e12bfc084832", package = "mp-felt", features = ["parity-scale-codec", "serde"]}
-mp-hashers = { git = "https://github.com/keep-starknet-strange/madara.git", rev = "f30acea8af7e28e956e771928130e12bfc084832", package = "mp-hashers" }
-mp-transactions = { git = "https://github.com/keep-starknet-strange/madara.git", rev = "f30acea8af7e28e956e771928130e12bfc084832", package = "mp-transactions", features = ["parity-scale-codec", "serde"] }
+starknet-types-core = { version = "0.0.5", features=["hash", "parity-scale-codec"] }
```

Comment:
https://github.com/starknet-io/types-rs/issues/29

I'm afraid the feature `hash` of `starknet-types-core` is not no_sd compatible.

Reply:
You should add an `ensure_no_std` crate like I asked everyone to do across the ecosystem, you can find those in the cairo-vm repository, or our froks of blockifier. 
https://github.com/lambdaclass/cairo-vm/tree/main/ensure-no_std

Add a build of this crate in the CI.

Reply:
Right now your lib compile, but it will conflict when imported in an actual wasm binary.
That's why the`ensure_no_std` crate is usefull.

Reply:
Thank you for your feedback. I wasn't aware of those guidelines, we will implement this as soon as we solve the `hash` problem.

Are you aware of any no_std-compatible Pedersen hash implementations that we should use ? Because this is a blocking point

Reply:
starknet-rs one is.
Look into `madara/crates/primitives/hashers/src/pedersen.rs` to see how we use it. You probably will have to reimplement it manually tho, because you are using your own felt type. 
Or you can just wait for upstream to fix their `hash` impl. 
no_std is non-blocking for us rn, so there is a bit of room

Reply:
I opened a PR to add no_std upstream: https://github.com/lambdaclass/lambdaworks/pull/734
Once merged we will bump `starknet-types-core` and then `hash` should be no_std compatible.
So you just have to wait a bit